### PR TITLE
Updated deploy yamls with following changes

### PR DESCRIPTION
**Pull request by vishnoianil**
> Use NEXAPI prefix for env var of apiserver
> User subdomain (try.nexodus.io) as a frontend url

Also,

> Fix integration tests with new urls
> Updated references  nexodus.local to try.nexodus.local

### CODE REVIEW

**Comment by vishnoianil** [comment]
FYI..@Gregory-Pereira

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -324,7 +324,7 @@ jobs:
 
       - name: Add hosts to /etc/hosts
         run: |
-          echo "127.0.0.1 auth.nexodus.local api.nexodus.local nexodus.local" | sudo tee -a /etc/hosts
+          echo "127.0.0.1 auth.try.nexodus.local api.try.nexodus.local try.nexodus.local" | sudo tee -a /etc/hosts
```
Comment (russellb) [inline .github/workflows/build.yml:327]: I would also be fine changing this back. We only need the added subdomain for prod.

**Review by russellb — APPROVED** [review]
I’m good with this. You could have left the local domain as-is, but it’s easy to change later too.

File: .github/workflows/build.yml (hunk 1 above)
Comment (vishnoianil) [reply .github/workflows/build.yml:327]: yeah I thought of that, but just for consistency sake, i changed it locally as well. Hopefully it won't create confusion when it comes to configuration of deployments (locally and prod/qa).

**Pull request merged**